eap-xp4/runtime-image/rel-jdk17-overrides.yaml

```diff
@@ -0,0 +1,42 @@
+schema_version: 1
+
```

Comment:
Forget my comments related to runtime image, I am seeing the override now.

Reply:
About runtime image, maybe is better:
1) move the runtime image for XP to root folder like EAP
mv eap-xp4/runtime-image runtime-image-xp
2) remove duplicated
rm -rf eap-xp4-jdk17/runtime-image

So the tree should be:
tree -d -L 1
.
├── eap-xp3
├── eap-xp4
├── eap-xp4-jdk17
├── jdk17-image
├── modules
├── runtime-image
├── runtime-image-xp
└── tests

Reply:
Add @xstefank to review

Reply:
@jfdenise What do you think about this?

Reply:
@rdnovell , I am not sure, I would keep the runtime image inside the eap-xp4 directory and use an override. It could create confusion for xp3 image. What do you think?

Reply:
Oh yes, I forgot the XP3

Reply:
Again just curious, I'm not that involved in the project, but since XP 3 is out of support scope, do we need to keep it in the master branch?

Reply:
I think we can tag or create a branch for XP3 and then remove from master branch. @jfdenise What do you think about that?